app/src/main/java/com/github/libretube/ui/activities/AboutActivity.kt

```diff
@@ -28,6 +30,12 @@ class AboutActivity : BaseActivity() {
         binding.toolbar.setNavigationOnClickListener {
             onBackPressedDispatcher.onBackPressed()
         }
+        // manually apply additional padding for edge-to-edge compatibility
+        ViewCompat.setOnApplyWindowInsetsListener(binding.toolbar) { v, insets ->
+            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
+            v.setPadding(0, systemBars.top, 0, 0)
+            insets
+        }
```

Comment:
Considering nearly the same code is used in many screens, we should move it to the base activity.